runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -242,7 +242,7 @@ public NettyClientRouter(String host, Integer port, Boolean tls,
         clientID = UUID.randomUUID();
         connected = false;
         timeoutConnect = 500;
-        timeoutResponse = 5000;
+        timeoutResponse = 5000* 10;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)